src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/IdealResponseSaver.php

```diff
@@ -19,7 +19,7 @@ class IdealResponseSaver extends AbstractResponseSaver
      */
     public function save(QuoteTransfer $quoteTransfer): QuoteTransfer
     {
-        $responseTransfer = $quoteTransfer->getPayment()->getComputopIdeal()->getIdealInitResponse();
+        $responseTransfer = $quoteTransfer->getPaymentOrFail()->getComputopIdealOrFail()->getIdealInitResponseOrFail();
```

Comment:
`getHeaderOrFail` should be used below
API spec needs to be updated
same for all `*RespoinseSaver` classes

Reply:
still not fixed

Reply:
Fixed